solutions/vertexai/ai-experimentation/Kptfile

```diff
@@ -0,0 +1,16 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: ai-sandbox-project
+  annotations:
+    config.kubernetes.io/local-config: "true"
+info:
+  description: |
+    Landing zone v2 subpackage.
+    Depends on `experimentation/core-landing-zone`.
```

Comment:
I think this should read experimentation/client-landing-zone .... like in the README.md